rllib/models/tf/tf_distributions.py

```diff
@@ -110,8 +108,21 @@ def __init__(
         logits: tf.Tensor = None,
         temperature: float = 1.0,
     ) -> None:
+        # We assert this here because to_deterministic makes this assumption.
```

Comment:
Before, the torch distribution made this assumption. I'm adding this code because this class makes the same assumption now.

Reply:
What if both are None? Then the assert would pass.
Better:
```
assert (probs is None) != (logits is None)
```